InstanceExport/PowerShell/README.md

```diff
@@ -8,17 +8,17 @@ In order to use it, you should:
 
 2. Reach out to us at help@devresults.com to request an Instance Export Manifest.
 
-3. Save the manifest file in the same directory/folder you have saved the powershell script. It's important to save the file in the JSON format and that you remember what you named the file. In this tutorial the name we used is *manifest.json*.
+3. Save the manifest file in the same directory/folder you have saved the powershell script. It's important to save the file in the JSON format and that you remember what you named the file. In this tutorial the name we used is _manifest.json_.
```

Comment:
Note that the screenshot uses a slightly different file name; update?

Reply:
I agree that it's probably worth syncing this up with the screenshot, or taking a new screenshot.

Reply:
I've updated the images of the README.md file. Hope all is good now.